index.rst

```diff
@@ -2,24 +2,54 @@
+Alysha Shugart and Tiago Ribeiro were executing some dome movement tests with the remote help of Bruno Quint.
 
-This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
+At one point around 8:30 PM local time, Tiago Ribeiro went to update LOVE in the love02 node with the latest version of the logging system for testing during the night.
+The node was completely shutdown (no LOVE system or ospl daemon running), shortly after the ospl daemon on love2 was initialized the network outage started.
+We were able to communicate with La Serena over Slack using a poor cellular connection and after being able to raise IT support it was established that it was impossible to recover from the issue remotely.
+
+At that point the crew decided to call in the night.
+The Auxiliary Telescope dome was manually closed and the telescope was left in the position it was in.
+The mirror covers were closed and all equipment was left in a safe state.
+
+The next morning IT arrived at the summit and determined that the issue was caused because two hosts were broadcasting using the same IP, which created an internal loop, causing the ACI to completely shutdown the network.
+They were able to determined that the IP responsible for the incident was 139.229.170.30, from the main ospl daemon, running on azar1, and the ospl daemon that was started on love2, initiating the episode.
+
+Mitigation Strategies
+=====================
+
+In summary, the key factors behind this incident are:
+
+* CISCO ACI network.
+* Brittle manual configuration required by DDS + docker
+
+In terms of mitigation strategies, it is worth noting that the CISCO ACI is already scheduled to be deprecated in early 2023.
+
+It is also worth noting that we are planning to retire DDS in favor of Kafka in the first quarter of 2023.
+Kafka does not use multicast traffic and requires a much simpler setup, with no need to setup daemons.
 
-Add content here
-================
+Altogether, the two main risk factors involved in this incident are already planned to be deprecated in a short time frame.
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Nevertheless, additional mitigation could be performed in case the Kafka adoption is delayed.
+In some nodes, we have successfully deployed the ospl daemon using the host network.
+The migration to using the host network in all the azar and love nodes would require some help from IT in setting up the correct network interface.
+This would eliminate the need to manually specify the network address, removing the possibility of further collisions in the future.
+Given the current timeline for adoption of Kafka, we are not sure this task is worth the effort.
```

Comment:
Maybe add the timeline of the ACI replacement as well, as it was part of the problem?  You did mention it above.
Kind of surprised there is only one mitigation that can be performed.  Is there not some procedural system that would keep this from happenings (such as requiring that someone else check the IP addresses).  Also surprised that no one on the summit can fix this - can we train the Observing Specialist in how to shut down nodes if this happens again - that way we won't lose the entire night.

Reply:
I don't think making the process more complicated would help much. The problem is that when this happened the entire network went down. There was no way to login to any computer, even the internal WiFi and wired networks went down. 

I think that without the ACI, most likely the network would survive but the system at hand would be disconnected from the network. I am honestly not sure if the entire host would be kicked off or just the container. But honestly, the best solution would be to use the host network such that we don't have to manually specify IP addresses.